langlib/lang.annotations/src/main/ballerina/annotations.bal

```diff
@@ -65,7 +65,7 @@ const annotation ServiceIntrospectionDocConfig IntrospectionDocConfig on service
 # The usage of a deprecated program element is not recommended due to
 # various reasons. Hence, the compiler issues a warning when such an element is used.
 public const annotation deprecated on source type, source class, source const, source annotation,
-                source function, source parameter, source object function, source object field;
+                source function, source parameter, source object function, source object field, source record field;
```

Comment:
Can't seem to find the previous comment related to this. Let's use

```suggestion
                source function, source parameter, source object function, source field;
```

Reply:
Noted, I will update it